cpp/inc/bond/core/config.h

```diff
@@ -40,3 +40,9 @@
 #define BOND_CONSTEXPR          BOOST_CONSTEXPR
 #define BOND_CONSTEXPR_OR_CONST BOOST_CONSTEXPR_OR_CONST
 #define BOND_STATIC_CONSTEXPR   BOOST_STATIC_CONSTEXPR
+
+#ifdef BOND_HEADER_ONLY
```

Comment:
We should augment our documentation about [optimizing C++ build time](https://microsoft.github.io/bond/manual/bond_cpp.html#optimizing-build-time) (or similar) to mention this new macro.

Reply:
Updated.